Remove serde_bytes from dev-dependency, which in turn dev-depends on bincode
and creates circular dependency.
--- a/Cargo.toml	1970-01-01 08:00:00.000000000 +0800
+++ b/Cargo.toml	2019-06-03 17:14:36.454016457 +0800
@@ -28,8 +28,6 @@
 
 [dependencies.serde]
 version = "^1.0.63"
-[dev-dependencies.serde_bytes]
-version = "^0.10.3"
 
 [dev-dependencies.serde_derive]
 version = "^1.0.27"
